### DISCUSSION TITLE
Running OpenRadioss on windows sytem with AMD Ryzen CPU.

### DISCUSSION THREAD

**Discussion by ychoi777** [OpenRadioss/OpenRadioss#2217] [Q&A] [answered]
Hellow, 
I am try to install and run OpenRadioss on the windows 10 with AMD Ryzen CPU.
The OpenRadioss_win64.zip file is ready for windows with Intel's MPI program.

OpenRadioss on my Ryzen machine is not work with Intel's MPI program.
I guess that Microsoft's MPI(MS-MPI) for windows 10 or 11 is alternative one.

Could you help me for the problem ?

**Comment by MarianBulla** [comment]
Hello ychoi777,
thanks for your interest on OpenRadioss.

May I ask you, if you may try the OpenRadioss version, without MPI, first ?

Thanks and best regards,
Marian

**Reply by ychoi777** [reply]
Hello, Marian

I cannot find the OpenRadioss without MPI (for windows).
Could you tell me where to download the OpenRadioss without MPI (for windows). ?

Thanks and best regards
Young Choi

**Reply by MarianBulla** [reply]
Hello ychoi777,

please have a look here: https://github.com/OpenRadioss/OpenRadioss/releases

You will find all the precompiled executables, including version without MPI for Linux and Windows.
There is also a tutorial video on YouTube on how to use it.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by ychoi777** [comment, marked as answer]
Thanks.
I can run OpenRadioss_smp on my windos 11 with Ryzen Machine.

Thanks and best regards,
Young Choi
